vpr/src/pack/re_cluster_util.h

```diff
@@ -32,52 +33,73 @@ ClusterBlockId atom_to_cluster(const AtomBlockId& atom);
 std::vector<AtomBlockId> cluster_to_atoms(const ClusterBlockId& cluster);
 
 /**
- * @brief A function that loads the router data for a cluster
+ * @brief A function that loads the intra-cluster router data of one cluster
  */
 t_lb_router_data* lb_load_router_data(std::vector<t_lb_type_rr_node>* lb_type_rr_graphs,
-                                      const ClusterBlockId& clb_index);
+                                      const ClusterBlockId& clb_index,
+                                      const std::vector<AtomBlockId>& clb_atoms);
 
 /**
- * @brief A function that removes an atom from a cluster and check legality of
+ * @brief A function that removes a molecule from a cluster and check legality of
  * the old cluster. 
  *
  * It returns true if the removal is done and the old cluster is legal.
  * It aborts the removal and returns false if the removal will make the old cluster 
  * illegal
```

Comment:
Good to explain what data structures are updated (can be high-level) to indicate the new clustering state.